Draft for wallpapers call (#430)
haiku/website: content/news/2020-09-28-wallpapers-contest.md

+++
type = "news"
title = "Call for Wallpapers"
author = "bitigchi"
date = "2020-10-01 13:00:00+03:00"
tags = ["wallpapers", "contest"]
+++

> One of the most common feedback that we receive is that the Haiku needs some nice colourful wallpapers shipping with Haiku releases. Suffice to say, that we agree with this completely. We definitely want to improve the current situation. And we need your help.

Reviewer: I suppose doing the contest, implies we do agree.

```suggestion
One of the most common feedback that we receive is that Haiku needs some nice colourful wallpapers shipping with its releases. We definitely want to improve the current situation. And we need your help.
```

> Haiku currently ships with only one (1) wallpaper, and it's not a complete wallpaper per se, it's just the Haiku logo positioned on the screen on an appropriate position according to the screen resolution. To be fair, there is a reason we only use the Haiku logo as the default wallpaper; we want Haiku to be usable on systems that have memory as low as 256 MB, therefore each byte that we can spare counts.

Reviewer: I think the positioning of the logo isn't even resolution dependend...

```suggestion
Haiku currently ships with only one (1) wallpaper, and it's not a complete wallpaper per se, it's just the Haiku logo positioned at fixed coordinates on the screen. To be fair, there is a reason we only use the Haiku logo as the default wallpaper; we want Haiku to be usable on systems that have memory as low as 256 MB, therefore each byte that we can spare counts.
```

Author: I have noticed that the logo position is different on different systems. Sometimes it's on the centre, sometimes it's close to the bottom. How does it work?

Reviewer: Are you sure those systems ran the same resolution? AFAIK, the logo appears at fixed X/Y coordinates, origin being the top left corner of the screen. The higher the resolution the further the logo moves up and left.

We are looking for artists that can contribute to Haiku with some appealing wallpapers. To be more precise, we are looking for either:

- Vector illustrations
- Painted illustrations

## Guidelines

> Artists are free to submit in any category they please. We hope below guidelines will help you in creating the new Haiku wallpapers:

Reviewer: ```suggestion
Artists are free to submit in any category they please. We hope the guidelines below will help:
```

Author: Fixed. For some reason commit suggestion got greyed out.

- Illustrations should be in at least landscape 5K size.

> - Vector illustrations should include a "lite" version, to be used as the default wallpaper (see the second paragraph). Color scheme of this lite variant should blend in with the standard system background colour.

Reviewer: ```suggestion
- Vector illustrations should include a "lite" version, to be used as the default wallpaper (see the second paragraph). The colour scheme of this lite variant should blend in with the standard system background colour.
```

Maybe quickly summarize what's in the 2nd paragraph. Do you mean "(as a logo placed on the default background)" ?

- We encourage painted illustrations to use subtle haiku themes, such as leaves, Far-Eastern typography, and literature. Anime-themed drawings are also welcome, but should there be human characters, they should not be the main focus.

> - For vector illustrations, use darker blue tones as the main colour. For painted illustrations, use warmer colours.

Reviewer: I don't see why a vector graphic shouldn't have warm colours, or bitmaps have dark blue ones. It's weirdly specific... :)

Author: I just thought we would like to keep the blue tones that we use today. For drawings, it should be easy on the eyes. Moreover, all the Far-Eastern drawings I've encountered had warmer colours and natural elements.

Reviewer: I just don't see the distinction made between vector and bitmap. I do see that a logo that's supposed to overlay the standard blue background has to fit in, ofc.

Author: They are two different categories. Bitmap can be anything, but the vector is supposed to be a new default background, expanding on the current theme. That's why we need a vector lite version.

Or we can remove all constraints, let artists do what they please, and implement a file selector according to the RAM and resolution.

- Doodle versions of the Haiku logo are welcome, in-fact we would love to have a coloured Haiku logo doodle for the default wallpaper image (see the part about the lite version).
- Entries should include a note that the work is licensed under [CC Attribution-ShareAlike 4.0 International](https://creativecommons.org/licenses/by-sa/4.0/) or a similar license.
- Artists are encouraged to name their work as they please, otherwise you would be leaving the naming part to the mercy of the Haiku developers.

## Submission deadline

> There is no strict deadline for the call, we would like you to take as much time as you need. We are planning to release Beta3 in the first quarter of 2021, we would love to include your work by then. Depending on the number of entries before the release date, a vote might be held at the Haiku forums in each respective category.

Reviewer: I don't think we should mention a timeframe for Beta3. We are wont to slip those... Maybe just:
"We would love to include your work in the next (beta) release"

What are those mentioned categories, vector and bitmap? If so, why even distinguish those. If not, the categories should be stated explicitely somewhere in the text.

We should also mention the file format for the submissions. We need the raw working files, think Illustrator files, for example for vector graphics, Photoshop files for bitmaps etc. Plus SVG / PNG files?

Author: We should require WonderBrush format! :P
